distrax/_src/distributions/gumbel_test.py

```diff
@@ -0,0 +1,247 @@
+        attribute_string='cdf',
+        dist_args=distr_params,
+        call_args=(value,),
+        assertion_fn=self.assertion_fn)
+
+  @chex.all_variants
+  @parameterized.named_parameters(
+      ('1d dist, 1d value', (0, 1), 1),
+      ('1d dist, 2d value', (0.5, 0.1), np.array([1, 2])),
+      ('1d dist, 2d value as list', (0.5, 0.1), [1, 2]),
+      ('2d dist, 1d value', (0.5 + np.zeros(2), 0.3 * np.ones(2)), 1),
+      ('2d broadcasted dist, 1d value', (np.zeros(2), 0.8), 1),
+      ('2d dist, 2d value', ([0.1, -0.5], 0.9 * np.ones(2)), np.array([1, 2])),
+      ('1d dist, 1d value, edge case', (0, 1), 200),
+  )
+  def test_log_cdf(self, distr_params, value):
+    distr_params = (np.asarray(distr_params[0], dtype=np.float32),
+                    np.asarray(distr_params[1], dtype=np.float32))
+    value = np.asarray(value, dtype=np.float32)
+    super()._test_attribute(
+        attribute_string='log_cdf',
+        dist_args=distr_params,
+        call_args=(value,),
+        assertion_fn=self.assertion_fn)
+
+  @chex.all_variants(with_pmap=False)
+  @parameterized.named_parameters(
+      ('entropy', ([0., 1., -0.5], [0.5, 1., 1.5]), 'entropy'),
+      ('entropy broadcasted loc', (0.5, [0.5, 1., 1.5]), 'entropy'),
+      ('entropy broadcasted scale', ([0., 1., -0.5], 0.8), 'entropy'),
+      ('mean', ([0., 1., -0.5], [0.5, 1., 1.5]), 'mean'),
+      ('mean broadcasted loc', (0.5, [0.5, 1., 1.5]), 'mean'),
+      ('mean broadcasted scale', ([0., 1., -0.5], 0.8), 'mean'),
+      ('variance', ([0., 1., -0.5], [0.5, 1., 1.5]), 'variance'),
+      ('variance broadcasted loc', (0.5, [0.5, 1., 1.5]), 'variance'),
+      ('variance broadcasted scale', ([0., 1., -0.5], 0.8), 'variance'),
+      ('stddev', ([0., 1., -0.5], [0.5, 1., 1.5]), 'stddev'),
+      ('stddev broadcasted loc', (0.5, [0.5, 1., 1.5]), 'stddev'),
+      ('stddev broadcasted scale', ([0., 1., -0.5], 0.8), 'stddev'),
+      ('mode', ([0., 1., -0.5], [0.5, 1., 1.5]), 'mode'),
+      ('mode broadcasted loc', (0.5, [0.5, 1., 1.5]), 'mode'),
+      ('mode broadcasted scale', ([0., 1., -0.5], 0.8), 'mode'),
+  )
+  def test_method(self, distr_params, function_string):
+    distr_params = (np.asarray(distr_params[0], dtype=np.float32),
+                    np.asarray(distr_params[1], dtype=np.float32))
+    super()._test_attribute(
+        attribute_string=function_string,
+        dist_args=distr_params,
+        assertion_fn=self.assertion_fn)
+
+  @chex.all_variants(with_pmap=False)
+  @parameterized.named_parameters(
+      ('no broadcast', ([0., 1., -0.5], [0.5, 1., 1.5])),
+      ('broadcasted loc', (0.5, [0.5, 1., 1.5])),
+      ('broadcasted scale', ([0., 1., -0.5], 0.8)),
+  )
+  def test_median(self, distr_params):
+    distr_params = (np.asarray(distr_params[0], dtype=np.float32),
+                    np.asarray(distr_params[1], dtype=np.float32))
+    dist = self.distrax_cls(*distr_params)
+    self.assertion_fn(self.variant(dist.median)(), dist.median())
```

Comment:
Replace `dist.median()` with the actual value of the median, which can be manually computed here - that will ensure that the result has the correct value.